src/tests/functional/plugin/cpu/subgraph_tests/src/conv_sum_broadcast.cpp

```diff
@@ -0,0 +1,252 @@
+            auto localPrc = inpNode->get_element_type();
+            ngraph::Shape newShape = generatePerChannelShape(inpNode);
+            return ngraph::builder::makeFakeQuantize(inpNode, localPrc, 256, newShape);
+        }, "FakeQuantize(PerChannel)"}}), {"FakeQuantize"} };
+
+const auto fusingSigmoidFQFQ = fusingSpecificParams{ std::make_shared<postNodesMgr>(std::vector<postNodeBuilder>{
+        {[](std::shared_ptr<ngraph::Node> inpNode, const ngraph::element::Type& ngPrc, ngraph::ParameterVector& params){
+            return ngraph::builder::makeActivation(inpNode, ngPrc, ngraph::helpers::Sigmoid);
+        }, "Sigmoid"},
+        {[](std::shared_ptr<ngraph::Node> inpNode, const ngraph::element::Type& ngPrc, ngraph::ParameterVector& params){
+            auto localPrc = inpNode->get_element_type();
+            ngraph::Shape newShape = generatePerChannelShape(inpNode);
+            return ngraph::builder::makeFakeQuantize(inpNode, localPrc, 256, newShape);
+        }, "FakeQuantize(PerChannel)"},
+        {[](std::shared_ptr<ngraph::Node> inpNode, const ngraph::element::Type& ngPrc, ngraph::ParameterVector& params){
+            auto localPrc = inpNode->get_element_type();
+            ngraph::Shape newShape = generatePerChannelShape(inpNode);
+            return ngraph::builder::makeFakeQuantize(inpNode, localPrc, 256, newShape);
+        }, "FakeQuantize(PerChannel)"}}), {"Sigmoid", "FakeQuantize", "FakeQuantize"} };
+
+const auto fusingClampFQ = fusingSpecificParams{ std::make_shared<postNodesMgr>(std::vector<postNodeBuilder>{
+        {[](std::shared_ptr<ngraph::Node> inpNode, const ngraph::element::Type& ngPrc, ngraph::ParameterVector& params){
+            return ngraph::builder::makeActivation(inpNode, ngPrc, ngraph::helpers::Clamp, {}, {3.0f, 6.0f});
+        }, "Clamp"},
+        {[](std::shared_ptr<ngraph::Node> inpNode, const ngraph::element::Type& ngPrc, ngraph::ParameterVector& params){
+            auto localPrc = inpNode->get_element_type();
+            ngraph::Shape newShape = generatePerChannelShape(inpNode);
+            return ngraph::builder::makeFakeQuantize(inpNode, localPrc, 256, newShape);
+        }, "FakeQuantize(PerChannel)"}}), {"FakeQuantize"} };
+
+
+
+const std::vector<fusingSpecificParams> fusingParamsSet{
+        emptyFusingSpec,
+        fusingSigmoid,
+        fusingFakeQuantizePerTensorRelu,
+        fusingFakeQuantizePerChannelRelu,
+        fusingFQPerChannelSigmoidFQPerChannel,
+        fusingReluScaleShift,
+        fusingMulAddFQMullAdd,
+        fusingSigmoidFQFQ,
+//        fusingClampFQ // TODO: we need investigation, this particular pattern does not work even in static case
+        fusingDivSubFQ
+};
+
+const std::vector<fusingSpecificParams> fusingParamsSetBF16{
+        emptyFusingSpec,
+        fusingSigmoid,
+        fusingReluScaleShift
+};
+
+InputShape convInpShape = {
```

Comment:
as I can see here we test only cases when we broadcast second sum input to conv output
it would be great to test following cases:
1. broadcast output conv to second sum input. i.e. conv out [1, 32, 1, 1], sum in [1, 32, 64, 128]
2. broadcast out and second input to each other. i.e. conv out [1, 32, 1, 128], sum in [1, 32, 64, 1]

Reply:
I've added {1,32,3,10},{1,64,8,1} case so that the two inputs broadcast each other.